src/store/modules/order/actions.ts

```diff
@@ -27,6 +30,22 @@ const actions: ActionTree<OrderState, RootState> = {
     }
     return resp;
   },
+
+  // Add Product information in cache
+
+  async getProductInformation(context, { orders }){
```

Comment:
Remove duplicate declaration of the same function. You have defined the `getProductInformation` here in the order action and also in the product action.

Reply:
Okay sir will update